src/playdate/scoreboards.nim

```diff
@@ -31,11 +31,16 @@ type
         lastUpdated*: uint32
         boards*: seq[PDBoard]
 
-    PDResultKind* = enum PDResultSuccess, PDResultError
+    PDResultKind* = enum 
+      PDResultSuccess, 
+      PDResultSuccessEmpty
```

Comment:
Bike shedding: it’s not really a “success”. Maybe ‘PDResultUnavailable’? ‘PDResultIncomplete’?

Reply:
I like Unavailable 👍